ddhcpd/README.md

```diff
@@ -0,0 +1,32 @@
+# DDHCPD
+
+
+## Integration of DDHCPD in gluon
+
+Add the package `ddhcpd` in your `site.mk` and add a section in your `site.conf`:
```

Comment:
Add the feed https://github.com/sargon/gluon-sargon to your 'modules' config, add the package `ddhcpd` to your `site.mk` and add a section in your `site.conf`: